src/main/kotlin/no/nav/klage/oppgave/eventlisteners/StatistikkTilDVHService.kt

```diff
@@ -111,7 +111,7 @@ class StatistikkTilDVHService(
             saksbehandlerEnhet = klagebehandling.tildeling?.enhet,
             tekniskTid = klagebehandling.modified,
             vedtakId = klagebehandling.currentDelbehandling().id.toString(),
-            vedtaksdato = klagebehandling.avsluttetAvSaksbehandler?.toLocalDate(),
+            vedtaksdato = klagebehandling.currentDelbehandling().avsluttetAvSaksbehandler?.toLocalDate(),
```

Comment:
Dette vil ikke bli riktig for anker, men det er det nok for klager.

Reply:
Ja, denne opererer kun på klagebehandlinger per nå.